aten/src/ATen/native/native_functions.yaml

```diff
@@ -10226,7 +10226,7 @@
   dispatch:
     CPU, CUDA: linalg_matrix_exp
 
-- func: linalg_matrix_exp_backward(Tensor self, Tensor grad) -> Tensor
+- func: linalg_matrix_exp_backward(Tensor self, Tensor grad, bool backward_ad) -> Tensor
```

Comment:
I'm not a fan of the function being called `*_backward` and then having a flag to choose between forward and backward mode.
But `*_backward()` functions are special cased not to be dumped in the `torch.*` namespace. So I think it is ok.

Reply:
I was going to call it `_differential`  and call that boolean `adjoint`, but I chose to keep things simple in the end.

Reply:
We might want to update the codegen to handle `_jvp` and `_differential` functions the same way we special case `_backward` so that we can easily do these renaming in the future.